languages/en/dictionary.js

```diff
@@ -0,0 +1,13 @@
+var read = require('fs').readFileSync;
+var dictiaonary = null;
+
+module.exports = function() {
+    if (dictiaonary ===  null) {
+        var base = require.resolve('dictionary-en-us');
+        dictiaonary = {
+            'aff': read(base.replace('.js', '.aff'), 'utf-8'),
```

Comment:
Could you explain why do we need to perform such gymnastics when loading the dictionary, instead of simply requiring it and passing it over to `nspell`, [as described in their usage example](https://www.npmjs.com/package/nspell#usage)?

Also, there is a typo in the spelling of the "dictionary" variable.

Reply:
Fixed the typo.

Regarding "gymnastics": the dictionary can be loaded on an async fashion. But the whole sentiment library works synchronously. So I decided to not pollute the whole library with callbacks or promises, and given that I couldn't use async/await to do it (eslint complained about it) I decided to load it this way.